test/integration/compiler.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { Expect, Test, TestCase } from "alsatian";
+import { execSync } from "child_process"
+import { existsSync, removeSync, unlink } from "fs-extra"
```

Comment:
Why do you need `fs-extra` all those functions exist in default `fs`.

Reply:
`fs-extra` can handle non empty directories.

Reply:
RIP thought default node could do that by now.